Swift/Function.swift

```diff
@@ -39,8 +39,12 @@ public final class Function {
     ///
     /// - Parameter expression: The expression.
     /// - Returns: The COUNT(expr) function.
-    public static func count(_ expression: ExpressionProtocol) -> ExpressionProtocol {
-        return QueryExpression(CBLQueryFunction.count(expression.toImpl()))
+    public static func count(_ expression: ExpressionProtocol? = nil) -> ExpressionProtocol {
+        if let expression = expression {
+            return QueryExpression(CBLQueryFunction.count(expression.toImpl()))
+        } else {
+            return QueryExpression(CBLQueryFunction.count(Expression.property("*").toImpl()))
```

Comment:
If expression is nil, nil can be passed to Objective-C count directory.

```
return QueryExpression(CBLQueryFunction.count(expression != nil ? expression.toImpl() : nil))
```

Reply:
Need to update the `+ (CBLQueryExpression*) count: (CBLQueryExpression*)expression;` signature to allow null as well.